build/deploy/main.go

```diff
@@ -0,0 +1,105 @@
+// main handles deployment of the plugin to a development server using either the Client4 API
+// or by copying the plugin bundle into a sibling mattermost-server/plugin directory.
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+
+	"github.com/mattermost/mattermost-server/model"
+	"github.com/mholt/archiver/v3"
+	"github.com/pkg/errors"
+)
+
+func main() {
+	err := deploy()
+	if err != nil {
+		log.Print(err.Error())
```

Comment:
Maybe we change this to 
```suggestion
		log.Printf("Failed to deploy plugin: %s" + perr.Error())
```
and modify the other error messages to start with a lower case latter? This would be golint compliant.